src/webgpu/gpu_test.ts

```diff
@@ -187,7 +187,9 @@ export class GPUTest extends Fixture {
       const check1 = this.checkBuffer(actual.subarray(begin, end), expected1);
       const check2 = this.checkBuffer(actual.subarray(begin, end), expected2);
       if (check1 !== undefined && check2 !== undefined) {
-        niceStack.message = check1 + check2;
+        niceStack.message = <span class="x x-first x-last">`Expected one of the following two checks to succeed:</span>
+- ${check1}
+- ${check2}`;
```

Comment:
@kainino0x , do you know why this code snippet failed to build?

Reply:
I have no idea where the span stuff came from?? It must have been a GitHub bug.